components/FavoriteButton/useToggleFav.tsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from "react";
```

Comment:
this useTobbleFav does not match my code in `reco/#7-favorites-button`

Reply:
I initially had your code, but I tweaked it slightly in terms of parameters and where some checks were.